omnij-rpc/src/main/java/foundation/omni/rpc/OmniExtendedClient.java

```diff
@@ -103,10 +104,10 @@ public Sha256Hash acceptDexOffer(Address fromAddress, CurrencyID currencyId, Big
     public Sha256Hash createMetaDexSellOffer(Address address, CurrencyID currencyForSale, BigDecimal amountForSale,
                                       CurrencyID currencyDesired, BigDecimal amountDesired,
                                       Byte action) throws JsonRPCException, IOException {
-        Long willetsForSale = BTC.btcToSatoshis(amountForSale).longValue();  // Assume divisible property
-        Long willetsDesired = BTC.btcToSatoshis(amountDesired).longValue();  // Assume divisible property
+        Coin willetsForSale = BTC.btcToCoin(amountForSale);  // Assume divisible property
```

Comment:
To my understanding this should be `OmniDivisibleValue`, and it must be `OmniDivisibleValue`, if `Coin` has a limit smaller than `63^2-1`, same for L64, and L108 (but not L65, L66, which really refer to BTC).